test/plugins/TestInvalidQmlPlugin.qml

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
So weird for GitHub to say this is a moved file. One was deleted in #324 and the other was added in #315